frontend/fingertips-frontend/components/organisms/BenchmarkLegend/index.tsx

```diff
@@ -0,0 +1,72 @@
+'use client';
+
+import { BenchmarkLabel } from '@/components/organisms/BenchmarkLabel/index';
+import styled from 'styled-components';
+import { LabelText } from 'govuk-react';
+
+const DefaultBenchmarkLegendGroupPanelStyle = styled('div')({
+  'alignItems': 'center',
+  'alignContent': 'center',
+  'marginBottom': '0.23em',
+  '& .legend_panel_header': {
+    border: '0px',
+    margin: '0px 0px 0px 1px',
+    alignSelf: 'stretch',
+    color: '#000',
+    fontSize: '16px',
+    fontStyle: 'normal',
+    fontWeight: '300',
+    lineHeight: '22px',
+  },
+});
+
+export interface BenchmarkData {
+  title?: string;
+  group: string;
+  types: string[];
+}
+
+export const Benchmark = () => {
```

Comment:
I think this should probably be called `BenchmarkLabel`:
```suggestion
export const BenchmarkLabel = () => {
```

Reply:
This is the main component with many labels.  I agreed a suitable name should be given, instead of Benchmark , should I called it BenchmarkLabelPanel ? since this component can be reuse.
![image](https://github.com/user-attachments/assets/b1b16a4f-0666-49da-bda1-c36fa0e07778)

Reply:
or maybe BenchmarkLegend